_posts/2017-10-20-totw-55.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Tip of the Week #55: Name Counting and unique_ptr"
+layout: tips
+sidenav: side-nav-tips.html
+permalink: tips/55
+publish: true
+type: markdown
+---
+
+Originally posted Sep 12, 2013
+
+*by Titus Winters [(titus@google.com)](mailto:titus@google.com)*
+
+Updated Oct 20, 2017
+
+Quicklink: **abseil.io/tips/55**
```

Comment:
This doesn't seem to linkify

Reply:
GH Pages doesn't like autolinking to same document. I added a manual link to self.